### PR TITLE
Add "sonarpedia.json", update rule metadata

### PR DESCRIPTION
**Pull request by christophe-zurn-sonarsource**

### CODE REVIEW

File: sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/AvoidCommentedOutCodeCheck.json [hunk 1]
```diff
@@ -10,5 +10,7 @@
     "misra",
     "unused"
   ],
-  "defaultSeverity": "Major"
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-125",
+  "sqKey": "S125"
```
Comment (pynicolas) [inline sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/AvoidCommentedOutCodeCheck.json]: It should be "AvoidCommentedOutCodeCheck". Same thing for other rules.

File: sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/FileLengthCheck.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Files should not have too many lines",
+  "title": "Files should not have too many lines of code",
```
Comment (pynicolas) [inline sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/FileLengthCheck.json]: I think that the implementation still count lines and not "lines of code".

**Comment by christophe-zurn-sonarsource** [comment]
Updated metadata following comments

File: sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/ComplexityCheck.json [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
     "linearFactor": "1min"
   },
   "tags": [
-    "brain-overload"
+    
   ],
-  "defaultSeverity": "Major"
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-1908",
+  "sqKey": "S1908"
```
Comment (pynicolas) [inline sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/ComplexityCheck.json]: Should be "ComplexityCheck".

File: sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/HeaderCheck.json [hunk 4]
```diff
@@ -9,5 +9,7 @@
   "tags": [
     
   ],
-  "defaultSeverity": "Blocker"
+  "defaultSeverity": "Blocker",
+  "ruleSpecification": "RSPEC-1451",
+  "sqKey": "S1451"
```
Comment (pynicolas) [inline sonar-web-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/HeaderCheck.json]: Should be "HeaderCheck"

**Comment by christophe-zurn-sonarsource** [comment]
Thank's for the keys I missed :)

**Pull request merged**